src/pages/components/Navbar/DeveloperNavbar.js

```diff
@@ -0,0 +1,81 @@
+import {
```

Comment:
question: why are you creating a brand new navbar? these are the same options that we have in our existing navbar - couldn't we reuse the navbar that we already have?

Reply:
I actually wanted to use the existing nav bar.

However the Figma design is a bit different from it and I thought it was intentional from the design to make that of the developer dashboard different from the main nav bar.

Reply:
hmm, let's confirm with David if we can update the Figma designs so they match our current navbar, or if he wants to update our existing navbar to match his newer designs

i think having two navbars at right now might be a bit much for us to maintain!